api/utils/ogc_translate.py

```diff
@@ -9,6 +9,10 @@
     "ows": "http://www.opengis.net/ows/2.0"
 }
 
+# This list extends the recommended WPS job statuses to include statuses 'Deduped', 'Deleted', and 'Offline'.
+WPS_STATUSES = ["Accepted", "Running", "Succeeded", "Failed", "Dismissed", "Deduped", "Deleted", "Offline"]
+
+HYSDS_STATUSES= ["job-started", "job-completed", "job-queued", "job-failed", "job-deduped", "job-revoked", "job-offline"]
```

Comment:
```suggestion
HYSDS_STATUSES= ["job-started", "job-completed", "job-queued", "job-failed", "job-deduped", "job-revoked", "job-offline"]
WPS_TO_HYSDS_JOB_STATUS_MAP = {
 "Accepted": "job-queued",
 "Running": "job-started",
 "Succeeded": "job-completed",
 "Failed": "job-failed",
 "Dismissed": "job-revoked",
 "Deduped": "job-deduped",
 "Deleted": None, 
 "Offline": "job-offline"}
```

What do you think about this change ? Its easier to follow than a long list of if, elifs. We can also change the hysds_to_ogc function to use a similar mapping in another ticket.